src/class-wpml-elementor-urls.php

```diff
@@ -23,6 +23,7 @@ public function __construct(
 
 	public function add_hooks() {
 		add_filter( 'elementor/document/urls/edit', array( $this, 'adjust_edit_with_elementor_url' ), 10, 2 );
+		add_filter( 'wpml_is_pagination_in_post', [ $this, 'is_pagination_in_post' ], 10, 3 );
```

Comment:
@mrudzki91 the filter's handle name is a bit confusing, in my opinion.

I had to look at the code and the implementation in the other MR to understand what it's supposed to do.

Can you please give it a better name?

The information I think are missing in the handle name are:

- Q: Which post are we talking about? A: The current one.
- Why pagination? This is term is usually used for the handling of the pagination (business logic), or for UI elements that handles the pagination. 
   Here instead we want to know if the (current) post is "paginated". 
   In fact, is not just the post, but the current URL/path of the current post (I believe because I'm still not that sure I understand how it works).

Perhaps something like `wpml_is_current_post_paginated`, even if it's still missing the information about the "URL", but adding that as well, would make the handle a bit too long.

Reply:
@andreasciamanna I've renamed the filter to `wpml_is_pagination_url_in_post` and also changed some variable names to make sure it's more clear.
This filter checks if the given URL is a pagination URL inside the given post (so it doesn't check the current post URL but some other URL inside the post `$post_name`).